packages/flutter/lib/src/widgets/tween_animation_builder.dart

```diff
@@ -29,8 +29,8 @@ import 'value_listenable_builder.dart';
 /// current animation value. The [builder] is called throughout the animation
 /// for every animation value until [Tween.end] is reached.
 ///
-/// A provided [onEnd] callback is called whenever an animation completes.
-/// Registering an [onEnd] callback my be useful to trigger an action (like
+/// A provided [ImplicitlyAnimatedWidget.onEnd] callback is called whenever an animation completes.
```

Comment:
I think we can move this comment to constructor and keep [onEnd]

Reply:
I am not sure what you mean...